src/pages/en/guides/cms/ghost.mdx

```diff
@@ -2,12 +2,287 @@
+
+Use the `posts` array returned from the fetch to display a list of blog posts on the page.
+
+```astro title="src/pages/index.astro"
+---
+import { api } from '../lib/api';
+const posts = await api.posts
+    .browse({
+        limit: 'all',
+    })
+    .catch((err) => {
+        console.error(err);
+    });
+---
+
+<html lang="en">
+    <head>
+        <title>Astro + Ghost 👻</title>
+    </head>
+    <body>
+
+        {
+            posts.map((post) => (
+                <>
+                    <a href={`/post/${post.slug}`}>
+                        <h1> {post.title} </h1>
+                    </a>
+                </>
+            ))
+        }
+    </body>
+</html>
+```
+
+### Generating pages
+
+The page `src/pages/post/[slug].astro` [dynamically generates a page](/en/core-concepts/routing/#dynamic-routes) for each post.
+
+<FileTree title="Project Structure">
+- src/
+ - lib/
+    - api.ts
+  - pages/
+    - index.astro
+    - post/
+      - **[slug].astro**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+Import the API to access blog posts using `posts.browse()` and return a post as props to each of your dynamic routes. 
+
+```astro title="src/pages/post/[slug].astro"
+---
+import { api } from '../../lib/api';
+
+export async function getStaticPaths() {
+    const posts = await api.posts
+        .browse({
+            limit: 'all',
+        })
+        .catch((err) => {
+            console.error(err);
+        });
+
+    return posts.map((post) => {
+        return {
+            params: {
+                slug: post.slug,
+            },
+            props: {
+                post: post,
+            },
+        };
+    });
+}
+
+const { post } = Astro.props;
+---
+```
+
+Create the template for each page using the properties of each `post` object.
+
+```astro title="src/pages/post/[slug].astro" ins={24-37}
+---
+import { api } from '../../lib/api';
+export async function getStaticPaths() {
+    const posts = await api.posts
+        .browse({
+            limit: 'all',
+        })
+        .catch((err) => {
+            console.error(err);
+        });
+    return posts.map((post) => {
+        return {
+            params: {
+                slug: post.slug,
+            },
+            props: {
+                post: post,
+            },
+        };
+    });
+}
+const { post } = Astro.props;
+---
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>{post.title}</title>
+    </head>
+    <body>
+        <img src={post.feature_image} alt={post.title} />
+
+        <h1>{post.title}</h1>
+        <p>{post.reading_time} min read</p> <!-- This property returns the estimated reading time in minutes -->
+
+        <Fragment set:html={post.html} />
+    </body>
+</html>
+```
+:::note
+`<Fragment />` is a built in Astro component which allows you to avoid an unnecessary wrapper element. This can be especially useful when fetching HTML from a CMS (e.g. Ghost or [WordPress](/en/guides/cms/wordpress/)).
```

Comment:
Missing hypen
```suggestion
`<Fragment />` is a built-in Astro component which allows you to avoid an unnecessary wrapper element. This can be especially useful when fetching HTML from a CMS (e.g. Ghost or [WordPress](/en/guides/cms/wordpress/)).
```